### ISSUE TITLE
Provide a pause button for Carousels

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#1112]
An example of a pause button was provided in v4 for the carousels (see https://boosted.orange.com/docs/4.6/components/carousel/#with-controls).

It is required for a11y purpose.

This example must be reintegrated in the version 5 of Boosted.

This issue maybe requires to specify with the designers where this pause button should be displayed. It could be integrated within the carousel and not necessarily outside the carousel.

If a JavaScript example must be present in the doc: transform the jQuery version of the v4 in JS in this v5.
